eth2util/hash_test.go

```diff
@@ -32,3 +32,13 @@ func TestSlotHashRoot(t *testing.T) {
 		hex.EncodeToString(resp[:]),
 	)
 }
+
+func TestHash(t *testing.T) {
+	hashOf0 := [32]byte{110, 52, 11, 156, 255, 179, 122, 152, 156, 165, 68, 230, 187, 120, 10, 44, 120, 144, 29, 63, 179, 55, 56, 118, 133, 17, 163, 6, 23, 175, 160, 29}
```

Comment:
same, if you get data or logic from somewhere else, you need to reference it. Else you need to explain how you pulled this out of thin air.

Reply:
Gotcha